moskito-extensions/moskito-disk-space-monitoring/src/main/java/net/anotheria/moskito/extensions/diskspacemonitoring/DiscSpaceMonitoringConfig.java

```diff
@@ -0,0 +1,57 @@
+package net.anotheria.moskito.extensions.diskspacemonitoring;
+
+
+import org.configureme.ConfigurationManager;
+import org.configureme.annotations.Configure;
+import org.configureme.annotations.ConfigureMe;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Config for DiscSpaceMonitoring plugin.
+ */
+@ConfigureMe(name = "disk-space-monitoring-plugin")
+public class DiscSpaceMonitoringConfig {
+    /**
+     * Configured disk list.
+     */
+    @Configure
+    private String[] disks;
+
+    /**
+     * Monitor object.
+     */
+    private static final Object monitor = new Object();
+
+    /**
+     * {@link DiscSpaceMonitoringConfig} instance
+     */
+    private static DiscSpaceMonitoringConfig instance;
```

Comment:
in this  case - volatile keyword required...
## description
http://www.cs.umd.edu/~pugh/java/memoryModel/DoubleCheckedLocking.html